cartographer/mapping_2d/scan_matching/ceres_scan_matcher.cc

```diff
@@ -60,14 +60,14 @@ CeresScanMatcher::CeresScanMatcher(
 CeresScanMatcher::~CeresScanMatcher() {}
 
 void CeresScanMatcher::Match(const transform::Rigid2d& previous_pose,
-                             const transform::Rigid2d& initial_pose_estimate,
+                             const transform::Rigid2d& local_pose_estimate,
```

Comment:
Let's drop the `local_` prefix here. It depends on the frame in which the `probability_grid` is expressed, right now the local SLAM frame in 2D, but the submap frame in 3D and we might want to change it in the future. Maybe we retain `initial_` here, but now in the meaning of starting point? Similar in the other scan matchers.